DSharpPlus.CommandsNext/Entities/CommandGroup.cs

```diff
@@ -58,9 +58,13 @@ public override async Task<CommandResult> ExecuteAsync(CommandContext ctx)
 
             if (cn != null)
             {
-                var cmd = ctx.Config.CaseSensitive
-                    ? this.Children.FirstOrDefault(xc => xc.Name == cn || (xc.Aliases != null && xc.Aliases.Contains(cn)))
-                    : this.Children.FirstOrDefault(xc => xc.Name.ToLowerInvariant() == cn.ToLowerInvariant() || (xc.Aliases != null && xc.Aliases.Select(xs => xs.ToLowerInvariant()).Contains(cn.ToLowerInvariant())));
+                var (comparison, comparer) = ctx.Config.CaseSensitive switch
+                {
+                    true  => (StringComparison.InvariantCulture, StringComparer.InvariantCulture),
+                    false => (StringComparison.InvariantCultureIgnoreCase, StringComparer.InvariantCultureIgnoreCase)
+                };
+                var cmd = this.Children.FirstOrDefault(xc =>
+                    xc.Name.Equals(cn, comparison) || (xc.Aliases != null && xc.Aliases.Contains(cn, comparer)));
```

Comment:
It's strange but it seems that this is dead code, I can't execute it. It seems this code is duplicated in https://github.com/DSharpPlus/DSharpPlus/blob/ba40ee3ef95bfb1d7a46b3f6f149b603593c6bb3/DSharpPlus.CommandsNext/CommandsNextExtension.cs#L288-L311 where it actually gets executed.

Code snippet is taken from latest commit in master branch.

Reply:
Taking a shot in the dark, I think that code is for finding and executing subcommands